libs/accounts/src/lib/accounts-data-provider.ts

```diff
@@ -50,3 +52,42 @@ export const accountsDataProvider = makeDataProvider<
   getData,
   getDelta,
 });
+
+export const getAccountData = (
+  data: AccountFieldsFragment[],
+  assetSymbol: string
+) => {
+  const assetData = data.filter((a) => a.asset.symbol === assetSymbol);
+
+  const deposited = assetData
+    .filter((a) => [AccountType.ACCOUNT_TYPE_GENERAL].includes(a.type))
+    .reduce((acc, a) => acc.plus(a.balance), new BigNumber(0));
+
+  const used = assetData
+    .filter((a) =>
+      [AccountType.ACCOUNT_TYPE_MARGIN, AccountType.ACCOUNT_TYPE_BOND].includes(
```

Comment:
Should we count in another type of accounts too or just MARGIN? (i.e. AccountType.ACCOUNT_TYPE_BOND, FEES, etc.)

Reply:
@campbellssource @candida-d